inventory/host_vars/openfoodnetwork.net/config.yml

```diff
@@ -25,3 +25,6 @@ postgres_listen_addresses:
 
 custom_hba_entries:
   - { type: hostssl, database: "{{ db }}", user: zapier, address: '54.86.9.50/32', auth_method: md5 }
+
+# Images settings
+attachment_path: ":rails_root/public/spree/products/:id/:style/:basename.:extension"
```

Comment:
this is already the default here:
https://github.com/openfoodfoundation/ofn-install/blob/0d4924ca6eb2f7811023afc49c2b860c15ca281f/roles/app/defaults/main.yml#L7